types/setting.go

```diff
@@ -1592,6 +1595,16 @@ var (
 		ReadOnly: false,
 		Default:  "false",
 	}
+
+	SettingDefinitionMaxPodRecreateBackoff = SettingDefinition{
+		DisplayName: "Maximum Pod Recreate Backoff",
+		Description: "Specifies the maximum duration (in seconds) to wait between pod recreation attempts. After each failed attempt, the backoff interval doubles until it reaches this limit. The default value is 120 seconds.",
+		Category:    SettingCategoryGeneral,
+		Type:        SettingTypeInt,
+		Required:    false,
+		ReadOnly:    false,
+		Default:     "120",
```

Comment:
```diff
+        ValueIntRange: map[string]int{
+			ValueIntRangeMinimum: 1,
+		},
```
I believe we do not allow a negative value here, right?

Reply:
yes, right. I've applied your suggestion.